README.md

```diff
@@ -8,31 +8,60 @@
 
 Warp 10 is an Open Source Geo Time Series Platform designed to handle data coming from sensors, monitoring systems and the Internet of Things.
 
-Geo Time Series extend the notion of Time Series by merging the sequence of sensor readings with the sequence of sensor locations. If your data have no location information, Warp 10 will handle them as regular Time Series.
+Geo Time Series extend the notion of Time Series by merging the sequence of sensor readings with the sequence of sensor locations.
+
+**If your data have no location information, Warp 10 will handle them as regular Time Series.**
 
 ## Features
 
-The Warp 10 Platform provides a rich set of features to simplify your work around sensor data.
+The Warp 10 Platform provides a rich set of features to simplify your work around sensor data:
 
 * Warp 10 Storage Engine, our collection and storage layer, a Geo Time Series Database
-* WarpScript, a language dedicated to sensor data analysis with more than 1000 functions and extension capabilities
+* [WarpLib](https://www.warp10.io/doc/reference), a library dedicated to sensor data analysis with more than 1000 functions and extension capabilities
+* [WarpScript](https://www.warp10.io/content/03_Documentation/04_WarpScript), a language specifically designed for analytics of time series data. It is one of the pillars of the analytics layer of the Warp 10 Platform
+* [FLoWS](https://www.warp10.io/content/03_Documentation/04_FLoWS), an alternative to WarpScript for users discovering the Warp 10 platform. It is meant to be easy to learn, look familiar to other programming languages and enable time series analysis by leveraging the whole of WarpLib.
```

Comment:
Platform.
look familiar to users of other programming languages

Reply:
done